github/release_notes/renderer.py

```diff
@@ -121,93 +123,102 @@ def _nodes_to_lines(
-                        # for the current repo we use gitHub's feature to auto-link to references,
-                        # hence in case of commits we don't need a prefix
-                        reference_prefix = ''
-                    if self._generate_link(rn_obj):
-                        reference_id_text = rn_obj.reference_id[0:12] # short commit hash
-
-                reference = '{reference_prefix}{ref_id}'.format(
-                    reference_prefix=reference_prefix,
-                    ref_id=reference_id_text,
-                )
+            header_suffix = ' ({s})'.format(
+                s=', '.join(header_suffix_list)
+            )
+        return header_suffix
 
-                if rn_obj.is_current_repo and not self._generate_link(rn_obj):
-                    header_suffix_list.append(reference)
-                else:
-                    if not self._generate_link(rn_obj):
-                        header_suffix_list.append(
-                            '{repo_path}{reference}'.format(
-                                repo_path=cn.github_repo_path(),
-                                reference=reference
-                            )
-                        )
-                    else:
-                        if rn_obj.reference_is_pr:
-                            reference_link = '{source_repo_url}/pull/{ref_id}'.format(
-                                source_repo_url=rn_obj.cn_source_repo.github_repo_url(),
-                                ref_id=rn_obj.reference_id
-                            )
-                        else: # commit
-                            reference_link = '{source_repo_url}/commit/{ref_id}'.format(
-                                source_repo_url=rn_obj.cn_source_repo.github_repo_url(),
-                                ref_id=rn_obj.reference_id
-                        )
+    def _header_suffix_reference(
+        self,
+        rls_note_obj: ReleaseNote
+    ):
+        cn_source_repo = rls_note_obj.cn_source_repo
+        reference_id_text = rls_note_obj.reference_id
 
-                        link_text = '{repo_path}{reference}'.format(
-                            repo_path=cn.github_repo_path(),
-                            reference=reference
-                        )
-                        header_suffix_list.append(
-                            self._build_link(url=reference_link, text=link_text)
-                        )
-            if rn_obj.user_login:
-                if not self._generate_link(rn_obj):
-                    header_suffix_list.append('@{u}'.format(
-                        u=rn_obj.user_login
-                    ))
-                else:
-                    user_link_text = '@{u}'.format(u=rn_obj.user_login)
-                    user_url = '{github_url}/{u}'.format(
-                         u=rn_obj.user_login,
-                        github_url=cn.github_url()
+        if rls_note_obj.reference_is_pr:
+            reference_prefix = '#'
```

Comment:
is there not an enum for that?

Reply:
that's the prefix that I was talking about in this TODO https://github.com/gardener/cc-utils/pull/100/files#diff-3eb5546c6806a652fd0452949bb2cf68R38

Reply:
issue addressed with 03b7d5a8262ef05b54b6dfcd545f0e8c1881adc2